layers/meta-resin-qemu/recipes-support/resin-runqemu/resin-runqemu.bb

```diff
@@ -0,0 +1,65 @@
+DESCRIPTION = "Resin helpers for running runqemu"
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://${RESIN_COREBASE}/COPYING.Apache-2.0;md5=89aea4e17d99a7cacdbeed46a0096b10"
+
+SRC_URI = " \
+    file://runqemu-resin \
+    file://run-dhcp.patch \
+    file://yes-reboot.patch \
```

Comment:
Because we want to reboot the target. There is a minimal comment in the patch file:

> Don't run qemu with -no-reboot for hddimg images as it works just fine.
> Tested on qemux86*.